### DISCUSSION TITLE
Netrc error on pod install

### DISCUSSION THREAD

**Discussion by dominikIncibit** [codemagic-ci-cd/codemagic-docs#1407] [Q&A] [answered]
My flutter project builds completely normally on my local machine (ios and android). 
When using codemagic though I keep getting this error in the build phase on pod install: 
`   Cloning into '/var/folders/w2/rrf5p87d1bbfyphxc7jdnyvh0000gn/T/d20230215-4422-7x13qa'...
         fatal: unable to access 'https://github.com/grpc/grpc.git/': .netrc parser error

Error running pod install`

Any idea on how to fix this?

**Comment by himesh-cm** [comment, marked as answer]
Hi @fleckdalm ,

Its a known issue with `xcode 14.2` Devs are looking into it, meanwhile, please use `Xcode 14.1` as the workaround.

**Reply by dominikIncibit** [reply]
Thank you for pointing this out. I will try to use Xcode 14.1 meanwhile and hope it gets fixed soon
